cas-maestro.php

```diff
@@ -16,308 +17,356 @@
+       * If setting hasn't been initialized (because original cas maestro plugin
        * had been installed, then set default value here
        */
-      if (empty($this->settings['phpcas_path'])) {
+      if ( empty( $this->settings['phpcas_path'] ) ) {
         $this->settings['phpcas_path'] = 'phpCAS/';
       }
-      include_once($this->settings['phpcas_path'] . 'CAS.php');
+      include_once( $this->settings['phpcas_path'] . 'CAS.php' );
 
-      if ($this->settings['server_hostname'] == '' ||
-          intval($this->settings['server_port']) == 0)
-          $this->cas_configured = false;
+      if ( $this->settings['server_hostname'] == '' ||
+          intval( $this->settings['server_port'] ) == 0 ) {
+        $this->cas_configured = false;
+      }
 
-      if ($this->cas_configured) {
-        //If everything is alright, let's initialize the phpCAS client
+      if ( $this->cas_configured ) {
+        // If everything is alright, let's initialize the phpCAS client
         phpCAS::client($this->settings['cas_version'],
-          $this->settings['server_hostname'],
-          intval($this->settings['server_port']),
-          $this->settings['server_path'],
-          false);
+            $this->settings['server_hostname'],
+            intval( $this->settings['server_port'] ),
+            $this->settings['server_path'],
+        false);
 
         // function added in phpCAS v. 0.6.0
         // checking for static method existance is frustrating in php4
         $phpCas = new phpCas();
-        if (method_exists($phpCas, 'setNoCasServerValidation'))
+        if ( method_exists( $phpCas, 'setNoCasServerValidation' ) ) {
           phpCAS::setNoCasServerValidation();
-        unset($phpCas);
+        }
+        unset( $phpCas );
         // if you want to set a cert, replace the above few lines
 
-        if ($this->settings['debug_path']) {
-          phpCAS::setDebug($this->settings['debug_path']);
+        if ( $this->settings['debug_path'] ) {
+          phpCAS::setDebug( $this->settings['debug_path'] );
         }
 
         /**
          * Filters and actions registration
          */
-        add_filter('authenticate', array(&$this, 'validate_login'), 30, 3);
-        add_filter('login_url', array(&$this, 'bypass_reauth'));
-        add_action('lost_password', array(&$this, 'disable_function'));
-        add_action('retrieve_password', array(&$this, 'disable_function'));
-        add_action('password_reset', array(&$this, 'disable_function'));
-        add_filter('show_password_fields', array(&$this, 'show_password_fields'));
+        add_filter( 'authenticate', array( &$this, 'validate_login' ), 30, 3 );
+        add_filter( 'login_url', array( &$this, 'bypass_reauth' ) );
+        // add_action('lost_password', array(&$this, 'disable_function'));
+        // add_action('retrieve_password', array(&$this, 'disable_function'));
+        // add_action('password_reset', array(&$this, 'disable_function'));
```

Comment:
Oops. I jumped the gun on issuing this PR. Those are just temporarily disabled until I work out a minor routing issue with non-CAS users. I described it in the specific commit, but obviously should have also added a TODO/FIXME comment.